darts/models/forecasting/ensemble_model.py

```diff
@@ -377,6 +382,126 @@ def _predictions_reduction(
             ]
         return predictions[0] if is_single_series else predictions
 
+    def save(
+        self, path: Optional[Union[str, os.PathLike, BinaryIO]] = None, **pkl_kwargs
+    ) -> None:
+        """
+        Saves the model under a given path or file handle.
+
+        Example for saving and loading a :class:`RegressionEnsembleModel`:
+
+            .. highlight:: python
+            .. code-block:: python
+
+                from darts.models import RegressionEnsembleModel, LinearRegressionModel, TiDEModel
+
+                model = RegressionEnsembleModel(
+                    forecasting_models = [
+                        LinearRegressionModel(lags=4),
+                        TiDEModel(input_chunk_length=4, output_chunk_length=4),
+                        ],
+                        regression_train_n_points=10,
+                )
+
+                model.save("my_model.pkl")
+                model_loaded = RegressionEnsembleModel.load("my_model.pkl")
+            ..
+
+        Parameters
+        ----------
+        path
+            Path or file handle under which to save the model at its current state. If no path is specified, the model
+            is automatically saved under ``"{ModelClass}_{YYYY-mm-dd_HH_MM_SS}.pkl"``.
+            E.g., ``"RegressionEnsembleModel_2020-01-01_12_00_00.pkl"``.
+        pkl_kwargs
+            Keyword arguments passed to `pickle.dump()`
+        """
+
+        if path is None:
+            # default path
+            path = self._default_save_path() + ".pkl"
+
+        if isinstance(path, (str, os.PathLike)):
+            # save the whole object using pickle
+            with open(path, "wb") as handle:
+                pickle.dump(obj=self, file=handle, **pkl_kwargs)
+        elif isinstance(path, io.BufferedWriter):
+            # save the whole object using pickle
+            pickle.dump(obj=self, file=path, **pkl_kwargs)
+        else:
+            raise_log(
+                ValueError(
+                    "Argument 'path' has to be either 'str' or 'PathLike' (for a filepath) "
+                    f"or 'BufferedWriter' (for an already opened file), but was '{path.__class__}'."
+                ),
+                logger=logger,
+            )
+
+        for i, m in enumerate(self.forecasting_models):
+            if issubclass(type(m), TorchForecastingModel):
+                # save the LightningModule checkpoint
+                path_ptl_ckpt = f"{path}.{type(m).__name__}_{i}.ckpt"
+                if m.trainer is not None:
+                    m.trainer.save_checkpoint(path_ptl_ckpt)
+                    print(path_ptl_ckpt + " is saved!")
```

Comment:
this should be a log right?

Reply:
Thank you for your review! @VascoSch92 
This print statement is the one I use for testing purposes and should be deleted here.